<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>243</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SOC CMD CONSOLE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SOC_Dialog.qrc">
    <normaloff>:/ICON/ICON.ico</normaloff>:/ICON/ICON.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(41, 41, 41);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 14pt &quot;Segoe UI Semibold&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string>REF_Location_Move</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="PB_GUIDE">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-image: url(:/BTN/resource/QuantumRED_MAIN/GUIDE_ON.png);
}
QPushButton:hover{
border-image: url(:/BTN/resource/QuantumRED_MAIN/GUIDE_ON_HOVER.png);
}
QPushButton:pressed{
border-image: url(:/BTN/resource/QuantumRED_MAIN/GUIDE_ON_PRESSED.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_GUIDE_SAVE">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-image: url(:/Button/resource/BUTTON/GUIDE_SAVE.png);
}
QPushButton:hover{
border-image: url(:/Button/resource/BUTTON/GUIDE_SAVE_HOVER.png);
}
QPushButton:pressed{
border-image: url(:/Button/resource/BUTTON/GUIDE_SAVE_PRESSED.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_MOVE_REF">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_19">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <spacer name="horizontalSpacer_15">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>2</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="PB_REF_UP_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(86, 86, 86);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="QuantumRED_MAIN.qrc">
                 <normaloff>:/ICON/resource/QuantumRED_MAIN/UP.png</normaloff>:/ICON/resource/QuantumRED_MAIN/UP.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_16">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>2</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_20">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="PB_REF_LEFT_2">
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(194, 194, 194);
background-color: rgb(86, 86, 86);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="QuantumRED_MAIN.qrc">
                 <normaloff>:/ICON/resource/QuantumRED_MAIN/LEFT.png</normaloff>:/ICON/resource/QuantumRED_MAIN/LEFT.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(66, 66, 66);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="PB_REF_RIGHT_2">
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(194, 194, 194);
background-color: rgb(86, 86, 86);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="QuantumRED_MAIN.qrc">
                 <normaloff>:/ICON/resource/QuantumRED_MAIN/RIGHT.png</normaloff>:/ICON/resource/QuantumRED_MAIN/RIGHT.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_21">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_22">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>2</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="PB_REF_DOWN">
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(194, 194, 194);
background-color: rgb(86, 86, 86);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="QuantumRED_MAIN.qrc">
                 <normaloff>:/ICON/resource/QuantumRED_MAIN/DOWN.png</normaloff>:/ICON/resource/QuantumRED_MAIN/DOWN.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_23">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>2</width>
                 <height>2</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="SOC_Dialog.qrc"/>
  <include location="QuantumRED_MAIN.qrc"/>
 </resources>
 <connections/>
</ui>
